services/examples/openlayers-wfs-static/index.html

```diff
@@ -96,7 +96,7 @@
         const workflow_id = json.id;
 
         console.log("Creating layer with workflow id:", workflow_id);
-
+        // TODO: add authentication header
```

Comment:
Is it now crashing?

Reply:
it won't work but it doesn't work in master right now either because of tokio